src/webgl-surface/buffers/static/uniform-ribbon-buffer.ts

```diff
@@ -232,8 +232,8 @@ export class UniformRibbonBuffer extends BaseBuffer <RibbonShape<any> | RibbonSh
 
         depth.x = ribbon.depth;
         resolution.y = MAX_SEGMENTS_PER_CURVE;
-        threshold.z = 10;
-        threshold.w = 10;
+        threshold.z = 50;
+        threshold.w = 50;
```

Comment:
50 per each end seems a little high. Did you try using lower numbers to see if they looked good enough?

Reply:
Also move the number 50 to a constant at the top of the file so we can change the number easily.